src/main/java/net/masterthought/cucumber/json/Step.java

```diff
@@ -1,22 +1,33 @@
 package net.masterthought.cucumber.json;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 
+import net.masterthought.cucumber.json.deserializers.StepDeserializer;
 import net.masterthought.cucumber.json.support.Resultsable;
 
+@JsonDeserialize(using = StepDeserializer.class)
 public class Step implements Resultsable {
 
-    // Start: attributes from JSON file report
-    private String name = null;
-    private final String keyword = null;
-    private final Result result = null;
-    private final Row[] rows = new Row[0];
-    private final Match match = null;
-    private final Embedding[] embeddings = new Embedding[0];
-    private final Output output = null;
-    @JsonProperty("doc_string")
-    private final DocString docString = null;
-    // End: attributes from JSON file report
+    private String name;
+    private final String keyword;
+    private final Result result;
+    private final Row[] rows;
+    private final Match match;
+    private final Embedding[] embeddings;
+    private final Output[] outputs;
```

Comment:
I returned previous implementation and created deserializer only for output node.